services/core-web/src/components/mine/ExplosivesPermit/MineExplosivesPermitTable.tsx

```diff
@@ -507,14 +678,31 @@ const MineExplosivesPermitTable: FC<RouteComponentProps & MineExplosivesPermitTa
       dataSource={transformRowData(data)}
       rowKey={(record) => record.explosives_permit_guid}
```

Comment:
I have a hunch that the rowKey prop isn't needed here or below on line 685- if the data has a key attribute, it will automatically be used, I think I saw it being set up above